cms/djangoapps/contentstore/utils.py

```diff
@@ -1173,25 +1162,6 @@ def gather_block_attributes(source_item, display_name=None, is_child=False):
     return duplicate_metadata, asides_to_create
 
 
-def load_services_for_studio(runtime, user):
```

Comment:
moved into services for reusability and reduce dependencies

Reply:
I suppose that keeping this (and `StudioPermissionsService`) here is fine. This code is used only in Studio, so keeping it in the `cms` package is correct. This is the approach we already use with other runtimes - for [LMS](https://github.com/openedx/edx-platform/blob/0b3e4d73b6fb6f41ae87cf2b77bca12052ee1ac8/lms/djangoapps/courseware/block_render.py#L484-L643) and [preview](https://github.com/openedx/edx-platform/blob/536493f261a8fbaaafecfec2226f374677c80a49/cms/djangoapps/contentstore/views/preview.py#L152-L251). 

Definitely +1 to reusing this function in `cms/djangoapps/contentstore/xblock_storage_handlers/view_handlers.py`, though.

Reply:
thanks for the suggestion! done